packages/nextjs/app/page.tsx

```diff
@@ -16,7 +22,8 @@ const Home: NextPage = () => {
           <p className="text-center text-lg">Get started by taking a look at your batch GitHub repository.</p>
           <p className="text-lg flex gap-2 justify-center">
             <span className="font-bold">Checked in builders count:</span>
-            <span>To Be Implemented</span>
+            <span className={`${!checkedInCounter.isFetched && "hidden"}`}>{Number(checkedInCounter.data)}</span>
+            <span className={`loading ${checkedInCounter.isFetched && "hidden"}`} />
```

Comment:
Checkout above comment maybe an easy, clean and more predictable way to handle to this would be : 

```tsx
  {checkedInCounter.isLoading ? (
    <span className="loading" />
  ) : (
    <span>{checkedInCounter?.data?.toString() || 0}</span>
  )}
```